dataactbroker/handlers/fileHandler.py

```diff
@@ -711,42 +704,18 @@ def upload_detached_file(self, create_credentials):
                 existing_submission = None
                 existing_submission_obj = None
 
-            for request_field, submission_field in request_job_mapping.items():
```

Comment:
Why are we removing all this? It might be fine, just walk me through why that decision was made.